demo_policy/e2e.bats

```diff
@@ -1,5 +1,58 @@
 #!/usr/bin/env bats
 
+@test "should allow pod creation in default namespace" {
```

Comment:
Please use a name that makes it clear this is a test for the `can-i` call. This makes it easier to understand what the test is about without looking into all the parameters.

Something like `should handle kubernetes can-i successful invocation`